examples/supply_chain/rl/algorithms/rule_based.py

```diff
@@ -5,9 +5,10 @@
 import math, random
 from typing import List, Optional
 
-from examples.supply_chain.rl.config import NUM_CONSUMER_ACTIONS, workflow_settings
 from maro.rl.policy import RuleBasedPolicy
 
+from ..config import NUM_CONSUMER_ACTIONS, workflow_settings
```

Comment:
Shall we use `from ..XXX` import? Or use absolute path to import?

Reply:
The absolute import method would cause 1 more Env instance creation here.
Maybe we can update the import method after the policy callback issue is solved?

Reply:
This won't be a problem after we launch `RLComponentBundle`